s2.asm

```diff
@@ -2293,88 +2335,99 @@ EniDec_ChkGetNextByte:
 
 ; For format explanation see http://info.sonicretro.org/Kosinski_compression
 ; ---------------------------------------------------------------------------
+
+; =============== S U B R O U T I N E =======================================
+
+
 ; KozDec_193A:
 KosDec:
-	subq.l	#2,sp
+	subq.l	#2,sp	; make space for two bytes on the stack
 	move.b	(a0)+,1(sp)
 	move.b	(a0)+,(sp)
-	move.w	(sp),d5
-	moveq	#$F,d4
+	move.w	(sp),d5	; copy first description field
+	moveq	#$F,d4	; 16 bits in a byte
 
--
-	lsr.w	#1,d5
+Kos_Decomp_Loop:
+	lsr.w	#1,d5	; bit which is shifted out goes into C flag
 	move	sr,d6
-	dbf	d4,+
+	dbf		d4,Kos_Decomp_ChkBit
```

Comment:
As s2.notes.txt says, the disassembly was set up with a tab size of 8 in mind, so this appears wrong.

Reply:
I though it looked fine but does that mean that ; =============== S U B R O U T I N E ======================================= appears wrong with 8 tabs ?

Reply:
Oh sorry I though window size not tabulation size sorry